### ISSUE TITLE
too much scrolling

### ISSUE THREAD

**Issue by gsam-7** [pulkithanda/navbars-hf#49]
if one has to look for a navbar, he/she has to to endless scrolling. my idea is to add a search box and instead of each navbar showing we can make different grids and assign new pages to them like this
![image](https://user-images.githubusercontent.com/68631593/140023019-59fdc392-35c3-4af5-9c5e-3794f2d562fb.png)

**Comment by pulkithanda** [comment]
Yes I plan to add a search box and filter so that people can find the navbars they want. Would you like to work on it? Or are you suggesting it as a feature improvement @gsam-7 ? Grids idea is great thanks for the heads up!

**Comment by gsam-7** [comment]
You can add the search box. I would like to work on the grid view

**Comment by gsam-7** [comment]
> You can add the search box. I would like to work on the grid view

Hi please assign me @pulkithanda

**Comment by pulkithanda** [comment]
Sure ok go ahead just I'd like to add hacktoberfest is over so this won't count there.

**Comment by gsam-7** [comment]
ye i know😢
